sequencer/call.go

```diff
@@ -0,0 +1,26 @@
+package sequencer
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/0xPolygon/cdk-data-availability/rpc"
+	"github.com/0xPolygon/cdk-data-availability/types"
+)
+
+// GetData returns batch data from the trusted sequencer
+func GetData(url string, batchNum uint64) (*types.Batch, error) {
+	response, err := rpc.JSONRPCCall(url, "zkevm_getBatchByNumber", batchNum, true)
```

Comment:
@arnaubennassar I am confused about this part. Is this the correct call for the trusted sequencer?

Reply:
In particular the CdkvalidiumSequenceBatches event seems like it can have more than one key in it, but it only has one NumBatch.